### PR TITLE
Exception handling

### PR DESCRIPTION
**Pull request by vldmrklnk**

### CODE REVIEW

File: Exadel.Compreface/Services/ExampleSubjectService.cs [hunk 1]
```diff
@@ -1,21 +1,54 @@
...
+
     public ExampleSubjectService(ComprefaceConfiguration configuration)
     {
         _configuration = configuration;
     }
-    
     public async Task<AddExampleSubjectResponse> AddExampleSubject(AddExampleSubjectRequest request)
+    {
+        try
+        {
```
Comment (asindarov) [inline Exadel.Compreface/Services/ExampleSubjectService.cs:29]: I would add Extra layer for Flurl.Http methods, to do that we need to create a static class like `ApiClient` or something. Then we will add `PostJsonAsync` method and other methods as well but in this case with `try/catch` so then we will not need to write `try/catch`  for every request, we will just use `ApiClient` class's extension methods to do the job

File: Exadel.Compreface/Exceptions/ServiceException.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+﻿namespace Exadel.Compreface.Exceptions
+{
+    public class ServiceException : Exception
+    {
+        public ServiceException(string _methodName, string _exceptionMessage) : base(ServiceExceptionMethod(_methodName, _exceptionMessage))
```
Comment (asindarov) [inline Exadel.Compreface/Exceptions/ServiceException.cs:5]: there is a standart way of creating custom user-defined exceptions, just follow this link 
https://learn.microsoft.com/en-us/dotnet/standard/exceptions/how-to-create-localized-exception-messages

**Review by asindarov — CHANGES_REQUESTED** [review]

File: Exadel.Compreface/Exceptions/ServiceException.cs (hunk 2 above)
Comment (vldmrklnk) [reply Exadel.Compreface/Exceptions/ServiceException.cs:5]: yeah, I know about it. But if we will use this custom example, we will need to write a lot of identical strings.

File: Exadel.Compreface/Services/ExampleSubjectService.cs (hunk 1 above)
Comment (vldmrklnk) [reply Exadel.Compreface/Services/ExampleSubjectService.cs:29]: Are you offer to write extensions for every method like a PostJsonAsync, GetJsonAsync and etc.?

**Pull request closed without merging**